### PR TITLE
less Order Review Page and components

### PR DESCRIPTION
**Pull request by elasticSzyszko**

### CODE REVIEW

File: src/components/checkout.summarylist.less [hunk 1]
```diff
@@ -24,6 +24,12 @@
 .cart-summary-list {
   padding: 0;
   
+  label {
```
Comment (dusanradovanovic) [inline src/components/checkout.summarylist.less]: Prefer a class name to an element name. With entire element styled if anyone ever adds another `label` to this component it'll get styled automatically.

File: src/components/ordertable.main.less [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        width: 100%;
+
+        tr {
+          display: block;
+          width: 100%;
+          border: 1px solid #eee;
+          margin-bottom: 30px;
+
+          td {
+            display: block!important;
```
Comment (dusanradovanovic) [inline src/components/ordertable.main.less]: Use `!important` only when overriding default Bootstrap styles. If you have to use it put a space before it.

File: src/containers/CartPage.less [hunk 3]
```diff
@@ -200,6 +200,69 @@
...
+          white-space: nowrap;
+          user-select: none;
+          border-radius: 2px;
+          margin-bottom: 2px;
+          color: #ffffff;
+          background-color: @mainColor;
+
+          &:focus {
+            outline: thin dotted #333;
+            outline: 5px auto -webkit-focus-ring-color;
```
Comment (dusanradovanovic) [inline src/containers/CartPage.less]: You can't define same css prop twice. One will get overridden. Pick one :) Also avoid vendor specific css. You probably need only first line.

File: src/containers/CartPage.less [hunk 4]
```diff
@@ -200,6 +200,69 @@
...
+            box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+          }
+
+          &.disabled,
+          &[disabled],
+          fieldset[disabled] & {
+            cursor: not-allowed;
+            pointer-events: none;
+            opacity: 0.65;
+            filter: alpha(opacity=65);
```
Comment (dusanradovanovic) [inline src/containers/CartPage.less]: You can remove this line. Regular `opacity` (previous line) is now supported in all browsers.

File: src/containers/CartPage.less [hunk 5]
```diff
@@ -200,6 +200,69 @@
...
+            -webkit-box-shadow: none;
```
Comment (dusanradovanovic) [inline src/containers/CartPage.less]: Not needed.

File: src/containers/OrderReviewPage.less [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+          white-space: nowrap;
+          user-select: none;
+          border-radius: 2px;
+          margin-bottom: 2px;
+          color: #ffffff;
+          background-color: @mainColor;
+
+          &:focus {
+            outline: thin dotted #333;
+            outline: 5px auto -webkit-focus-ring-color;
```
Comment (dusanradovanovic) [inline src/containers/OrderReviewPage.less]: Same as previous.

File: src/containers/OrderReviewPage.less [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+          &:focus {
+            color: #333333;
+            text-decoration: none;
+          }
+
+          &:active,
+          &.active {
+            outline: 0;
+            background-image: none;
+            -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
```
Comment (dusanradovanovic) [inline src/containers/OrderReviewPage.less]: Not needed.

File: src/containers/OrderReviewPage.less [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+            box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+          }
+
+          &.disabled,
+          &[disabled],
+          fieldset[disabled] & {
+            cursor: not-allowed;
+            pointer-events: none;
+            opacity: 0.65;
+            filter: alpha(opacity=65);
```
Comment (dusanradovanovic) [inline src/containers/OrderReviewPage.less]: Not needed.

File: src/containers/OrderReviewPage.less [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+            -webkit-box-shadow: none;
```
Comment (dusanradovanovic) [inline src/containers/OrderReviewPage.less]: Not needed.

**Comment by elasticSzyszko** [comment]
@dusanradovanovic addressed your comments

File: src/containers/CheckoutPage.less [hunk 10]
```diff
@@ -344,9 +344,9 @@
       }
       &:before,
       &:after {
-        content: " ";    
+        content: " ";
```
Comment (dusanradovanovic) [inline src/containers/CheckoutPage.less]: Indentation doesn't seem to be ok on these lines.

File: src/containers/OrderReviewPage.less (hunk 6 above)
Comment (dusanradovanovic) [reply src/containers/OrderReviewPage.less]: This is still present.

**Pull request merged**